apollo-router/src/warp_http_server_factory.rs

```diff
@@ -229,6 +231,19 @@ fn redirect_to_studio(host: Option<Authority>) -> Box<dyn Reply> {
     }
 }
 
+fn get_health_request() -> impl Filter<Extract = (Box<dyn Reply>,), Error = Rejection> + Clone {
+    warp::get()
+        .and(warp::path(".well-known"))
+        .and(warp::path("apollo"))
+        .and(warp::path("server-health"))
+        .and_then(move || async {
+            let mut result = HashMap::new();
+            result.insert("status", "pass");
+            let reply = Box::new(warp::reply::json(&result)) as Box<dyn Reply>;
+            Ok::<_, Rejection>(reply)
```

Comment:
[nit] You can move this allocation to a static [`Lazy`](https://docs.rs/once_cell/1.7.2/once_cell/sync/struct.Lazy.html)